.github/workflows/cd-release.yml

```diff
@@ -0,0 +1,53 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#
+# OpenCRVS is also distributed under the terms of the Civil Registration
+# & Healthcare Disclaimer located at http://opencrvs.org/license.
+#
+# Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+# graphic logo are (registered/a) trademark(s) of Plan International.
+name: Continuous Delievery And E2E Tests Performed On Staging
+
+on:
+  push:
+    branches:
+      - 'master'
```

Comment:
Perhaps it would be better to run it manually but I wanted to rarely merge to master from now on anyway and as our default branch is "develop", according to these docs, manual pipelines only work in the default branch so wouldnt work for master or releases: https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow. The Get COUNTRY_CONFIG_VERSION command is intended to get the version of the country config repo on its master.  I think that we could extend your script to manually run a deployment from develop if that was helpful that took in 2 Git hash values as secrets.

Reply:
The manual trigger uses the workflow file from the default branch, but we can still deploy from any branch as we're not deploying the source code itself, but the images we've built and pushed to Dockerhub. Agreed, we could have 2 different streams to push the deployments as we could then use the manual one for deploying fixes to releases that are currently being QA'd, but we don't yet want to push to master